backend/inspirehep/indexer/cli.py

```diff
@@ -334,3 +334,32 @@ def create_aliases(ctx, yes_i_know, prefix_alias):
             current_search.client.indices.delete_alias("*", alias_name)
         click.echo(f"Creating alias '{alias_name}' -> '{index_name}'")
         current_search.client.indices.put_alias(index=index_name, name=alias_name)
+
+
+@index.command(
+    "delete-indexes", help="Removes indexes with specified prefix",
+)
+@click.option("--yes-i-know", is_flag=True)
+@click.option("--prefix", default="", type=str)
+@with_appcontext
+@click.pass_context
+def create_aliases(ctx, yes_i_know, prefix):
+    indices = current_search.client.indices.get_alias()
+    prefix_regex = re.compile(f"""{prefix}.*""")
+    indices_to_delete = list(filter(prefix_regex.match, indices))
+    indices_to_delete = dict((k, indices[k]) for k in indices_to_delete)
+
+    if not indices_to_delete:
+        click.echo("No indices matching given prefix found.")
+        return 1
+    click.echo(f"""Found {len(indices_to_delete)} indices to delete""")
+    for index in indices_to_delete:
+        if not yes_i_know and not click.confirm(
+            f"This operation will remove '{index}' index. Are you sure you want to continue?"
+        ):
+            continue
+        aliases_to_delete = indices[index]["aliases"].keys()
+        for alias in aliases_to_delete:
+            current_search.client.indices.delete_alias(index=index, name=alias)
```

Comment:
Instead getting all aliases for index and then providing them here you can just use ```"*"``` as alias name and this will remove all aliases for provided index.

Reply:
Otherwise you can just provide all aliases as a string containing aliases separated by comma https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-delete-alias.html
As this loop will call ES separately for each alias.